### PR TITLE
Feature/publisher latch

### PR DESCRIPTION
**Pull request by alexemdesgagne**
## Description
Added some features:
- Some published messages are now latched.
- Alerts for leak sensor and battery levels.
- Dry run is using a latched message to allow persistence between telemetries.
- Thruster activate is using a latched message to allow persistence between telemetries.

Removed unused buttons in the toolbar.

## Fixes
Link all the related issues from the issue tracker
- Closes: #

## How has this been tested ?
I tested it locally.

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/components/Waypoints.tsx [hunk 1]
```diff
@@ -40,12 +40,11 @@ const Waypoints = () => {
     const [cmdSpeed, setCmdSpeed] = useState('0');
     const [cmdFine, setCmdFine] = useState('0.00');
     const [cmdMethod, setCmdMethod] = useState('0');
-    const [positionZ, setPositionZ] = useState('0.00');
```
Comment (Kam0ss) [inline src/components/Waypoints.tsx:43]: This deletion provokes an error.

File: src/components/Waypoints.tsx [hunk 2]
```diff
@@ -40,12 +40,11 @@ const Waypoints = () => {
...
-    
-    const setInitialPositionPublisher = useROSTopicPublisher<any>("/proc_simulation/start_simulation", "geometry_msgs/Pose");
-    const sendSingleAddPosePublisher = useROSTopicPublisher<any>("/proc_control/add_pose", "sonia_common/AddPose");
-    const sendMultipleAddPosePublisher = useROSTopicPublisher<any>("/proc_planner/send_multi_addpose", "sonia_common/MultiAddPose");
-    const resetTrajectoryPublisher = useROSTopicPublisher<any>("/proc_control/reset_trajectory", "std_msgs/Bool");
+
+    const setInitialPositionPublisher = useROSTopicPublisher<any>("/proc_simulation/start_simulation", "geometry_msgs/Pose", false);
+    const sendSingleAddPosePublisher = useROSTopicPublisher<any>("/proc_control/add_pose", "sonia_common/AddPose", true);
+    const sendMultipleAddPosePublisher = useROSTopicPublisher<any>("/proc_planner/send_multi_addpose", "sonia_common/MultiAddPose", true);
+    const resetTrajectoryPublisher = useROSTopicPublisher<any>("/proc_control/reset_traj", "std_msgs/Bool", false);
```
Comment (Kam0ss) [inline src/components/Waypoints.tsx]: The topic is /proc_control/reset_trajectory

**Review by Kam0ss — CHANGES_REQUESTED** [review]
There are errors when running the telemetry that must be fixed.

File: src/components/Waypoints.tsx (hunk 1 above)
Comment (alexemdesgagne) [reply src/components/Waypoints.tsx:43]: Yeah right my bad!

**Review by Kam0ss — APPROVED** [review]
LGTM

**Pull request merged**